app/src/main/java/org/fossasia/pslab/activity/WaveGeneratorActivity.java

```diff
@@ -68,17 +107,34 @@
     private String unit;
 
     //const values
+    public static final int FREQ_MIN = 10;
+    public static final int DUTY_MIN = 0;
+    public static final int PHASE_MIN = 0;
+
+    public static final int FREQ_MAX = 5000;
+    public static final int DUTY_MAX = 100;
+    public static final int PHASE_MAX = 360;
+
     public static final int SIN = 1;
```

Comment:
You can easily use Java `enum` for these kind of variables.

Reply:
If there is no other change could u plz merge it, I will address this changes in another PR as it is a lengthy task.

Reply:
If there are no other changes could u plz merge this PR as this enum task is lengthy and I will address that in another PR,

Reply:
Then can you open an issues and link it here so we have a record of you doing it in future?